tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml.cs

```diff
@@ -74,15 +74,24 @@ private async Task FillAvailableWidgetsAsync()
             return;
         }
 
-        var providerDefs = _hostingService.GetWidgetCatalog()!.GetProviderDefinitions();
-        var widgetDefs = _hostingService.GetWidgetCatalog()!.GetWidgetDefinitions();
+        var providerDefinitions = _hostingService.GetWidgetCatalog()!.GetProviderDefinitions();
+        var widgetDefinitions = _hostingService.GetWidgetCatalog()!.GetWidgetDefinitions();
 
-        Log.Logger()?.ReportInfo("AddWidgetDialog", $"Filling available widget list, found {providerDefs.Length} providers and {widgetDefs.Length} widgets");
+        if (providerDefinitions == null || widgetDefinitions == null)
+        {
+            return;
+        }
+
+        // Show the providers and widgets underneath them in alphabetical order.
+        var alphabeticalProviderDefinitions = providerDefinitions.ToList().OrderBy(x => x.DisplayName);

```

Comment:
NIT: `.OrderBy` before `.ToList()`

Unless the two are the same.  `.ToList()` then `.OrderBy()` looks like it'll make a list, then move things in that list.  However, I don't think this will be an issue until this list is 100+ units long.

Reply:
Ah, you can OrderBy an array. Neat.